bedrock/exp/templates/exp/firefox/whatsnew/whatsnew-fx70.html

```diff
@@ -0,0 +1,115 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros.html" import monitor_button, fxa_cta_link with context %}
+
+{% extends "exp/base/base-firefox.html" %}
+
+{% block page_title %}What’s new with Firefox - More privacy, more protections.{% endblock %}
+
+{# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
+{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
+
+{% block page_og_url %}{{ settings.CANONICAL_URL }}/{{ LANG }}/firefox/new/{% endblock %}
```

Comment:
Should this be `/firefox/70.0/whatsnew/`? Or possibly `firefox/{{ version }}/whatsnew/` to avoid hard-coding the version number?

Reply:
Never mind, while reading through https://github.com/mozilla/bedrock/pull/7908/ I realized the original whatsnew page is noindex, so I think pointing to /new is correct here.